Makefile

```diff
@@ -18,6 +18,8 @@ RELEASE_TAG = $(IMAGE_TAG)
 else
 IMAGE_TAG ?= $(RELEASE_TAG)
 
+RELEASE_DOC_TAG = `echo ${RELEASE_TAG} | awk -F "\." '{print $$1"."$$2}'`
```

Comment:
```suggestion
RELEASE_DOC_TAG = `echo ${RELEASE_TAG} | awk -F '.' '{print $$1"."$$2}'`
```

Reply:
Wierd, I didn't see a warning but maybe differences in versions? Either works for me (OSX) but will change

Reply:
ahh looks like BSD awk doesn't print the warning, but GNU awk does which is what the build env uses:
```
➜  ~ echo 'v0.26.0' | /usr/bin/awk -F"\." '{print $2}'
26
➜  ~ echo 'v0.26.0' | /usr/local/bin/gawk -F"\." '{print $2}'
gawk: warning: escape sequence `\.' treated as plain `.'
26
```

Reply:
```
➜  ~ /usr/bin/awk --version
awk version 20200816
➜  ~ /usr/local/bin/gawk --version
GNU Awk 5.1.1, API: 3.1 (GNU MPFR 4.1.0, GNU MP 6.2.1)
Copyright (C) 1989, 1991-2021 Free Software Foundation.
```